src/app/kbv/controllers/single-amount-question.js

```diff
@@ -15,15 +15,29 @@ class SingleAmountQuestionController extends BaseController {
         return callback(err, locals);
       }
 
+      if (req.session.question?.info?.months) {
+        locals.threeMonthsAgo = moment()
```

Comment:
Maybe `locals.dynamicDate` given the 3 months comes from the backend? (and the template doesn't care about this context)